.changeset/seven-books-relax.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/datastore": major
+"@fluidframework/test-utils": major
+---
+
+FluidDataStoreRuntime.getChannel throws for channels that do not exist
+
+Previously, calling `FluidDataStoreRuntime.getChannel(id)` for a channel that does not exist would wait for the channel to be created (possibly waiting indefinitely if never created). However, there is no safe means to dynamically create a channel in this manner without risking data corruption. The call will instead now throw for non-existent channels.
+
+TestFluidObject.load removed
```

Comment:
It looks like these are two separate changes. It's fine to have them in a single PR, but they should be in separate changesets. That way they show up as inidividual line items. E.g. imagine we create a rollup that just has the first line of each changeset - with a single changeset the "TestFluidObject.load removed" line would be missing in the rollup.

Of course all of this is changeable after merge but before release.

Reply:
This is a little evidence that maybe the CLI should prompt to describe _changes_, and create multiple changesets if needed...